rest/service/dashboardTemplate.go

```diff
@@ -20,6 +20,14 @@ var (
 			Scope:  "notifications",
 			Module: "./DashboardWidget",
 		},
+		models.LearningResources: models.ModuleFederationMetadata{
+			Scope:  "learningResources",
+			Module: "./BookmarkedLearningResourcesWidget",
```

Comment:
@karelhala FYI I fixed the typo in this PR. Let me know if you change the name to something else.